src/injectscripts/messages/create.js

```diff
@@ -0,0 +1,69 @@
+console.log("inject running");
+
+
+if (localStorage.getItem("LighthouseMessagesEnabled") == "true") {
+
+	whenWeAreReady(msgsystem,function() {
+
+	//Home HQ
+	whenWeAreReady(user,function() {
+		console.log("Setting Selected HQ to user HQ");
+		msgsystem.setSelectedHeadquarters(user.hq);
+	});
+
+	//Operational = true
+	msgsystem.operational(true);
```

Comment:
fills the send message page with all the contacts from the users LHQ. ill comment more in code